anrs/anrs-impl/src/main/java/com/duckduckgo/app/anr/AnrOfflinePixelSender.kt

```diff
@@ -45,7 +46,8 @@ class AnrOfflinePixelSender @Inject constructor(
                         ANR_WEBVIEW_VERSION to it.webView,
                     ),
                     mapOf(),
-                ).doOnComplete {
+                    DEFAULT,
+                ).ignoreElement().doOnComplete {
```

Comment:
Why do we need the `ignoreElement` here?

Reply:
the `sendPixel()` method now returns `Single<SendPixelResult>` instead of `Completable`, but in this particular case, we don't care about `SendPixelResult` and just want to know when the stream completes successfully, so ignoring that element converts `Single` back to `Completable`.